client/src/App.tsx

```diff
@@ -80,7 +80,13 @@ function App() {
           <Route path="budget" element={<Budget />} />
           <Route path="calendar" element={<Calendar />}/>
           <Route path="communication" element={<Communication />}/>
-          <Route path="settings" element={<SettingsPage />}/>
+          <Route path="settings" element={
+          <ProtectedRoute>
+            <ThemeProvider>
```

Comment:
The ThemeProvider wrapper is already present in the /dashboard path <Route /> component. Lines 83-89 shouldn't be necessary.

Reply:
ok removed